templates/chat/src/app/layout.tsx

```diff
@@ -0,0 +1,14 @@
+import './styles.css'
+
+export const metadata = {
+	title: 'tldraw chat starter kit',
```

Comment:
weird, i just noticed that `<title>` and this metadata is put at the bottom of the `body` tag, maybe only in dev? weird